db/seeds.rb

```diff
@@ -26,6 +26,20 @@ def puts_records_in_db(klass)
 
 # ---------------------------------------------
 
+@user_admin = User.where(email: 'admin@test.com').first_or_create!(
+  full_name: 'Admin User',
+  greeting_name: 'Addie',
+  language: 'spanish',
+  opt_in_email: false,
+  opt_in_text: false,
+  organization: 'Pie for Providers',
```

Comment:
We're using `ENV` to get the email and password for the 1 user created, should we use `ENV` here, too?
- or only use `ENV` for the admin
- or not use `ENV` at all?

Reply:
I'm gonna lean towards no ENVs, I think it makes it more confusing right now.